### ISSUE TITLE
Shaarli sends an absurdly high amount of shaarli_staySignedIn cookies

### ISSUE THREAD

**Issue by pikzen** [shaarli/Shaarli#84]
Note: this issue only exists when not logged in. The `shaarli_staySignedIn` cookie isn't even sent when logged in, which makes me think it's only sustaining through the session, but that's another issue.

On my local Shaarli installation with 12 links, Shaarli sends back a Set-Cookie header consisting of 20 times `shaarli_staySignedIn`, with the value as `deleted` and the expiration at timestamp 0 (January 1970)
![after](https://cloud.githubusercontent.com/assets/373126/5548998/955d8ad2-8b7f-11e4-82f7-0f7b31ebc121.png). (This screenshot is taken right after the offending commit, see below)

With only one link displayed, 9 of these cookies are sent.
Having tested on a remote server (in case using localhost would be the cause), the same problem arises.

The amount of cookies sent seems to be depending on the amount of links. (12 links => 20 cookies, 1 link => 9 cookies, 30 links => 67 cookies). I couldn't find a formula to explain why, but you can be pretty sure that somewhere Shaarli is concatenating stuff it shouldn't, and repeatedly.

This bug has appeared after commit ae00595 (A real "stay signed in": keep the connection) by @Sbgodin . Which makes it a bug that is over a year old :clap: :sparkler:
This is the network tab before the patch:
![before](https://cloud.githubusercontent.com/assets/373126/5549043/b766857e-8b80-11e4-9ab9-206f78bc5280.png)

tl;dr: it broke

EDIT: The complete Set-Cookie for a 30-link page. Wall of text.

```
"shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/shaarli_staySignedIn=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; path=/links/"
```

**Comment by nodiscc** [comment]
Pull request https://github.com/shaarli/Shaarli/pull/86 should fix this, needs a rebase and some testing.

**Comment by nodiscc** [comment]
Fixed in https://github.com/shaarli/Shaarli/pull/86
